components/kafka/src/main/kotlin/org/gxf/servicetemplate/kafka/GxfKafkaProducer.kt

```diff
@@ -14,7 +15,7 @@ import kotlin.random.Random
 
 @Service
 class GxfKafkaProducer(
-    private val kafkaTemplate: KafkaTemplate<String, Any>,
+    private val kafkaTemplate: KafkaTemplate<String, Measurement>,
```

Comment:
Voor KOD zullen we waarschijnlijk meerdere producers krijgen, nl. voor meet-, status- en alarmberichten (en mogelijk ook nog andere zoals bijv registratie- en configuratieberichten van domain naar protocol) 
Is Measurement dan voldoende dekkend?